README.md

```diff
@@ -31,14 +31,14 @@ require 'toopher_api'
 toopher = ToopherAPI.new("key", "secret")
 
 # Step 1 - Pair with their phone's Toopher app
-pairing = toopher.pair("pairing phrase", "username@yourservice.com")
+pairing = toopher.pair("username@yourservice.com", "pairing phrase")
 
 # Step 2 - Authenticate a log in
 auth_status = toopher.authenticate(pairing.id, 'my computer')
 
 # Once they've responded you can then check the status
 while auth_status.pending
-    auth_status = toopher.get_authentication_status(auth_status.id)
+    auth_status.refresh_from_server(toopher)
```

Comment:
While you're in the README, can you make it more consistent with the [new toopher-python README](https://github.com/toopher/toopher-python/blob/lang-library-refactor/README.md), please? For example, use `api` instead of `toopher` for the API object.